index.bs

```diff
@@ -1520,94 +1459,189 @@ If the attestation root certificate is not dedicated to a single WebAuthn Authen
+different versions are thus treated as different formats, e.g., `packed2` as a new version of the `packed` attestation statement
+format.
 
-## Packed Attestation Format ## {#packed-attestation}
+The following sections present a set of currently-defined and registered attestation statement formats and their identifiers.
+See the WebAuthn Attestation Format Identifier Registry defined in [[WebAuthn-Registries]] for an up-to-date list of registered
+attestation statement formats.
 
-Packed attestation is a WebAuthn optimized format of attestation data. It uses a very compact but still extensible encoding
-method. Encoding this format can even be implemented by <a>authenticators</a> with very limited resources (e.g., secure
-elements).
+## Packed Attestation Statement Format ## {#packed-attestation}
 
+This is a WebAuthn optimized attestation statement format. It uses a very compact but still extensible encoding method. Encoding
+this format can even be implemented by <a>authenticators</a> with very limited resources (e.g., secure elements).
 
 : Attestation format identifier
 :: packed
 
-
 : Attestation types supported
 :: All
 
 : Syntax
-:: A Packed Attestation statement has the following format:
-
-    <pre class="idl">
-        interface PackedAttestation {
-            readonly    attribute ArrayBuffer   x5c;
-            readonly    attribute ArrayBuffer   daaKey;
-            readonly    attribute DOMString     alg;
-            readonly    attribute ArrayBuffer   signature;
-        };
-    </pre>
+:: A Packed Attestation statement is a CBOR map with the following fields:
 
-    <div dfn-for="PackedAttestation">
-        The <dfn>x5c</dfn> attribute contains the attestation certificate and its certificate chain as described in [[!RFC7515]]
-        section 4.1.6.
+    : alg
+    :: A text string containing the name of the algorithm used to generate the attestation signature according to [[!RFC7518]]
+        section 3.1. The following algorithms are supported:
 
-        The <dfn>alg</dfn> element contains the name of the algorithm used to generate the attestation signature according to
-        [[!RFC7518]] section 3.1. The following algorithms are supported:
+        1. "ES256", "ES384" and "ES512" [[!RFC7518]]
+        2. "RS256", "RS384" and "RS512" [[!RFC7518]]
+        3. "PS256", "PS384" and "PS512" [[!RFC7518]]
+        4. "ED256" and "ED512" [[!FIDOEcdaaAlgorithm]]
 
-        1. "ES256" [[!RFC7518]]
-        2. "RS256" [[!RFC7518]]
-        3. "PS256" [[!RFC7518]]
-        4. "ED256" [[!FIDOEcdaaAlgorithm]]
-        5. "ED512" [[!FIDOEcdaaAlgorithm]]
+    : sig
+    :: A byte string containing the attestation signature.
 
-        The <dfn>signature</dfn> element contains the attestation signature.
-    </div>
+    : x5c
+    :: A definite-length array of byte strings. The elements of the array contain the attestation certificate and its
```

Comment:
why does this need to be a definite-length?  a CBOR decoder ought to transparently be able to handle both definite- and indefinite-length arrays at run time, yes?  Note that CDDL does not make any prescription as to whether arrays or maps use the definite length or indefinite length encoding.  see: https://tools.ietf.org/html/draft-greevenbosch-appsawg-cbor-cddl#section-3.2